interface/forms/telemed/view.php

```diff
@@ -0,0 +1,378 @@
+<?php
+/**
+ * transfer summary form.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Naina Mohamed <naina@capminds.com>
+ * @copyright Copyright (c) 2012-2013 Naina Mohamed <naina@capminds.com> CapMinds Technologies
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+
+require_once("../../globals.php");
+require_once("$srcdir/api.inc");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/options.inc.php");
+
+require_once("$srcdir/MedEx/API.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+use OpenEMR\OeUI\OemrUI;
+
+$MedEx = new MedExApi\MedEx('MedExBank.com');
+
+$form_name   = "telemed";
+$form_folder = "telemed";
+$Form_Name   = "TeleHealth Visit";
+$action      = $_REQUEST['action'];
+$id          = $_REQUEST['id'];
+$display     = $_REQUEST['display'];
+$pid         = $_SESSION['pid'];
+$refresh     = $_REQUEST['refresh'];
```

Comment:
In above code block, avoid using $_REQUEST. Use either $_GET or $_POST (whichever is being used for each variable).